src/components/body/scroller.component.ts

```diff
@@ -36,17 +36,21 @@ export class ScrollerComponent implements OnInit, OnDestroy {
   parentElement: any;
   onScrollListener: any;
 
-  constructor(private ngZone: NgZone, element: ElementRef) {
+  constructor(private ngZone: NgZone, element: ElementRef, private renderer: Renderer2) {
+
     this.element = element.nativeElement;
   }
 
   ngOnInit(): void {
     // manual bind so we don't always listen
     if (this.scrollbarV || this.scrollbarH) {
-      this.parentElement = this.element.parentElement.parentElement;
+      const renderer = this.renderer;
+      this.parentElement = renderer.parentNode(renderer.parentNode(this.element));
       this.ngZone.runOutsideAngular(() => {
-        this.parentElement.addEventListener('scroll', this.onScrolled.bind(this));
+      renderer.listen(
+        this.parentElement, 'scroll', this.onScrolled.bind(this));
```

Comment:
You actually shouldn't have to use the renderer to do this in v5.

Reply:
I originally started with v4